src/resources/deal21Cards.js

```diff
@@ -0,0 +1,8 @@
+const FIRST_CARD = 0;
+const DEAL_NUMBER = 21;
+
+function deal21Cards(deck) {
+  return [...deck].slice(FIRST_CARD, DEAL_NUMBER);
```

Comment:
O `slice` não altera o array, então não há necessidade de fazer o destructuring `[...deck]`.
```js
var list = [1, 2, 3, 4, 5, 6]

list.slice(0, 3)
=> [1, 2, 3]

list
=> [1, 2, 3, 4, 5, 6]
```

```suggestion
  return deck.slice(FIRST_CARD, DEAL_NUMBER);
```

Reply:
Ok! :fist_oncoming: